app/model/jobs.py

```diff
@@ -76,3 +80,7 @@ class Job(db.Model):
 
     def __repr__(self):
         return f"<{self.id}: {self.name}>"
+
```

Comment:
whats the reason of shortcuting words here ?
please chage to application* not appl
pretty hard to guess whats the context